### PR TITLE
ACMS-1205: Decouple Acquia CMS starter module.

### PR DESCRIPTION
**Pull request by vishalkhode1**
**Motivation**
Decouple Acquia CMS Starter module.

**Proposed changes**
- Updated Acquia CMS Starter module to remove content related to page/site_studio modules.
- Created new module: `acquia_cms_demo_content` to import site studio/page content.

**Alternatives considered**
N/A

**Testing steps**
- Manual Testing.
- PHPUnit tests should pass.

**Merge requirements**
- [ ] _Major change_
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Comment by vishalkhode1** [comment]
Will remove this [commit](https://github.com/acquia/acquia_cms/pull/1139/commits/19ce808084a1df66073f30d6c04234953f4e445a) after we test CI Job with full site install.

File: modules/acquia_cms_starter/modules/acquia_cms_demo_content/composer.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            "cweagans/composer-patches": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "drupal/core-composer-scaffold": true,
+            "ergebnis/composer-normalize": true,
+            "oomphinc/composer-installers-extender": true,
+            "phpro/grumphp-shim": true,
+            "webdriver-binary/binary-chromedriver": true
+        }
+    }
+}
```
Comment (chandan-singh7929) [inline modules/acquia_cms_starter/modules/acquia_cms_demo_content/composer.json:20]: @vishalkhode1 - I don't think there's any use of submodule composer.json all dependency should be added in starter composer.json itself.

**Comment by chandan-singh7929** [comment]
@vishalkhode1  As discussed, please create this PR against, starter module so that we can test this  in isolation with decoupled page CT.

**Comment by vishalkhode1** [comment]
> @vishalkhode1 As discussed, please create this PR against, starter module so that we can test this in isolation with decoupled page CT.

@chandan-singh7929 Created the PR. Following is the link: https://github.com/acquia/acquia_cms_starter/pull/3

**Comment by vishalkhode1** [comment]
Declining as PR will be merged from: https://github.com/acquia/acquia_cms_starter/pull/3

**Pull request closed without merging**